### ISSUE TITLE
Tracking over the jsgraph does not work ?

### ISSUE THREAD

**Issue by visualizer-feedback** [NPellet/visualizer#523]
Testcase: http://visualizer.epfl.ch/tiny/BcR6GECNH7myssaVID5l ([Original URL](http://visualizer.epfl.ch/c/home/index.html?viewURL=%2Fc%2Fd9bd75989166b0459041ad75bb6020b4%2Fview.json))

**Comment by targos** [comment]
you have to enable mouse tracking in the preferences

**Comment by lpatiny** [comment]
Thanks !

**Comment by andcastillo** [comment]
Still not working for 2D spectra

**Comment by lpatiny** [comment]
Don't think in was ever tested ...

**Comment by targos** [comment]
I don't think it is implemented for 2D spectra. Is it even possible ? jsGraph does not have the original data, we give it the computed contours

**Comment by NPellet** [comment]
jsGraph should give you an x y position in that case. Indeed it is not
possible to give a z value.

If it doesn't give an x y value, please open a ticket.

2015-04-30 9:21 GMT+02:00 Michaël Zasso <notifications@github.com>:
